api/Models/Hotel.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace iTechArt.Hotels.Api
+{
+    public class Hotel
+    {
+        public int HotelId { get; set; }
```

Comment:
But in `Account` class you've used `int Id`. Why did you choose different approach for `Hotel` class?